app/models/manageiq/providers/storage_manager/cinder_manager.rb

```diff
@@ -52,6 +52,10 @@ def self.description
     @description ||= "Cinder ".freeze
   end
 
+  def description
+    @description ||= "Cinder ".freeze
+  end
+
```

Comment:
Not a dup.  One is a class method and one is an instance  method.  The (new) instance method is required by the UI.